flake.nix

```diff
@@ -0,0 +1,148 @@
+{
+  description = "A flake for the KPlutus Semantics";
+
+  inputs = {
+    k-framework.url = "github:runtimeverification/k";
+    nixpkgs.follows = "k-framework/nixpkgs";
+    flake-utils.follows = "k-framework/flake-utils";
+    rv-utils.url = "github:runtimeverification/rv-nix-tools";
+    poetry2nix.follows = "pyk/poetry2nix";
+    blockchain-k-plugin.url =
+      "github:runtimeverification/blockchain-k-plugin/3010915ef37b0e614528821a7c8717c797a365a6";
+    blockchain-k-plugin.inputs.flake-utils.follows = "k-framework/flake-utils";
```

Comment:
```suggestion
    blockchain-k-plugin.url = "github:runtimeverification/blockchain-k-plugin/3010915ef37b0e614528821a7c8717c797a365a6";
    blockchain-k-plugin.inputs.flake-utils.follows = "k-framework/flake-utils";
```

Reply:
@goodlyrottenapple this has to be one-line or the sed expression in `.github/workflows/update-version.yml` needs to be updated.